README.md

```diff
@@ -32,9 +35,7 @@ Both approaches will use the same DocumentStore instance for all enabled feature
 configure.UsePersistence<RavenDB>(_ => _.UseDocumentStoreForTimeouts(...));
 ```
 
-## Wait I thought RavenDB was embedded in NServiceBus?
-
-So a little history
+## History
```

Comment:
Maybe as a separate document. For a readme on how to use the library, it's noise